apidoc/Titanium/App/iOS/iOS.yml

```diff
@@ -1617,3 +1712,83 @@ properties:
   - name: categories
     summary: Set of categories of user notification actions required by the applicaiton to use.
     type: Array<Titanium.App.iOS.UserNotificationCategory>
+
+---
+name: BackgroundTaskObject
+summary: |
+    Dictionary object of parameters used to register the background task using
+    the <Titanium.App.iOS.registerBackgroundTask> method.
+since: "10.0.0"
+platforms: [iphone, ipad, macos]
+
+properties:
+  - name: type
+    summary: Background task type.
+    type: String
+    constants: [Titanium.App.iOS.BACKGROUND_TASK_TYPE_REFRESH, Titanium.App.iOS.BACKGROUND_TASK_TYPE_PROCESS]
+  - name: identifier
```

Comment:
For legibility, I tend to add a newline between separate property/method/event entries in the yaml. Otherwise it makes it difficult to visually scan and see where each prop definition starts/ends.